hack/lib/golang.sh

```diff
@@ -433,6 +433,50 @@ kube::golang::get_physmem() {
   echo 1
 }
 
+kube::golang::unhide_godeps() {
+  echo "Restoring Godeps to their rightful location"
+  mv "${KUBE_ROOT}/.Godeps" "${KUBE_ROOT}/Godeps/"
+  rm "${KUBE_ROOT}/_output/local/go/vendor"
+}
+
+kube::golang::hide_godeps() {
+  echo "Hiding godeps from GB"
+  mv "${KUBE_ROOT}/Godeps" "${KUBE_ROOT}/.Godeps"
+  ln -s "${KUBE_ROOT}/.Godeps/_workspace" -T "${KUBE_ROOT}/_output/local/go/vendor" >/dev/null 2>&1 || true
+  trap kube::golang::unhide_godeps EXIT
+}
+
+kube::golang::verify_code_builds() {
+  if ! OUT=$(which gb 2>&1); then
+    hack/build-go.sh
+    return
+  fi
+  # Check for `go` binary and set ${GOPATH}.
+  kube::golang::setup_env
+
+  local alltargets
+  alltargets=("${KUBE_ALL_TARGETS[@]}")
+
+  local targets=()
+  for target in "${alltargets[@]}"; do
+    if [[ "${target}" == *".test" ]]; then
+      continue
+    fi
+    targets+=("${target}")
+  done
+  local binaries
+  binaries=($(kube::golang::binaries_from_targets "${targets[@]}"))
+
+  kube::log::status "Fast building the whole tree"
+  kube::log::status "  DO NOT USE THESE BUILDS FOR ANYTHING OTHER THAN A COMPILE CHECK"
+
+  kube::golang::hide_godeps
+
+  pushd "${KUBE_ROOT}/_output/local/go" > /dev/null
```

Comment:
does this work immediately after a `make clean` ?  I thought _output gets nuked.